src/main/groovy/life/qbic/samplestatus/reporter/services/QbicSampleTrackingService.groovy

```diff
@@ -93,112 +72,28 @@ class QbicSampleTrackingService implements SampleTrackingService {
     }
 
     @Override
-    void updateSampleLocation(String sampleCode, Location location, String status, Instant timestamp) throws SampleUpdateException {
-        String locationJson = DtoMapper.createJsonFromLocationWithStatus(location, status, timestamp)
-        HttpResponse<String> response = requestSampleUpdate(createSampleUpdateURI(sampleCode), locationJson)
+    void updateSampleStatus(String sampleCode, String status, Instant timestamp) throws SampleUpdateException {
+        String statusJson = DtoMapper.createJsonFromStatus(status, timestamp)
+        HttpResponse<String> response = requestSampleUpdate(createSampleUpdateURI(sampleCode), statusJson)
         if (response.statusCode() != 200) {
             throw new SampleUpdateException("Could not update $sampleCode to ${location.getLabel()} - ${response.statusCode()} : ${response.headers()}: ${response.body()}")
         }
     }
 
     private static class DtoMapper {
 
-        private static final LOCATION_NAME = "name"
-        private static final LOCATION_CONTACT = "responsible_person"
-        private static final LOCATION_CONTACT_EMAIL = "responsible_person_email"
-        private static final LOCATION_ADDRESS = "address"
-        private static final ADDRESS_AFFILIATION = "affiliation"
-        private static final ADDRESS_STREET = "street"
-        private static final ADDRESS_ZIP = "zip_code"
-        private static final ADDRESS_COUNTRY = "country"
-
-        protected static Optional<Location> parseLocationOfJson(String putativeLocationJson) {
-            println putativeLocationJson
-
-            List<Map> locationMaps = parseJsonToList(putativeLocationJson)
-            return locationMaps.stream().map(DtoMapper::convertMapToLocation).findFirst()
-        }
-
-        protected static String createJsonFromLocationWithStatus(Location location, String status, Instant arrivalTime) {
-            Map locationMap = convertLocationToMap(location)
-            locationMap.put("arrival_date", mapToLocationDateTimeString(arrivalTime))
-            locationMap.put("sample_status", status)
-            return JsonOutput.toJson(locationMap)
+        protected static String createJsonFromStatus(String status, Instant arrivalTime) {
+            Map parameters = new HashMap()
+            parameters.add("status" : status)
+            parameters.add("validFrom" : mapToDateTimeString(arrivalTime))
```

Comment:
```suggestion
            parameters.add("validFrom" : arrivalTime.toString())
```
As the new API circumvents the weird location time format, you can use the ISO format directly (provided by instant and automatically as UTC)